src/index.mako.html

```diff
@@ -464,7 +466,7 @@
                 </a>
               </p>
               <p>
-                <a ng-href="{{deviceSwitcherHref}}" translate>desktop_redirect</a>
+                <a href="{{deviceSwitcherHref}}" translate>desktop_redirect</a>
```

Comment:
just wondering: slow loading not affected? (If I recall correctly, we did it with ng-href so that link is not active as long as translation is not finished). Is this an update in 1.4?